config.js

```diff
@@ -91,6 +96,42 @@ const CONFIG = {
       updateUrl: 'http://teastop.plus.com/svn/teletext/',
     },
 
+    // [CONST.SERVICE_AMIGAROB]: {
```

Comment:
@peterkvt80 I've added the services that you have locally, but commented out the low quality ones - feel free to make your own changes here, but this feels like a good selection to serve.